tests/persons_tests.py

```diff
@@ -4,7 +4,21 @@
 class TestPersons(TestCase):
     def test_add_person_successfully(self):
         new_person = Persons()
-        added = new_person.add_person("Andy Carroll", "STAFF")
-        self.assertTrue(added)
-        self.assertEqual(new_person.name,"Andy Carroll")
-    #def test_add_person_invalid(self):
+        new_staff = new_person.add_person("Andy Carroll", "staff")
+        self.assertIsInstance(Staff)
```

Comment:
Is this correct? `self.assertIsInstance` takes two arguments -- an object and the class. Please review line 8.